diff --git a/grammar.js b/grammar.js
index 32ef7d0..9605b37 100644
--- a/grammar.js
+++ b/grammar.js
@@ -46,12 +46,21 @@ module.exports = grammar({
             choice(
                 $._plain_comment,
                 $._var_comment,
+                $._block_comment,
             ),
         _plain_comment: (_) =>
             seq(
                 COMMENT_PREFIX,
                 LINE_TAIL,
             ),
+        _block_comment: (_) =>
+            token(
+                prec(PREC.COMMENT_PREFIX, seq(
+                    "/*",
+                    /[^*]*\*+([^/*][^*]*\*+)*/,
+                    "/",
+                )),
+            ),
         _var_comment: ($) =>
             seq(
                 COMMENT_PREFIX,
@@ -116,7 +125,7 @@ module.exports = grammar({
                 prec.right(
                     repeat(
                         choice(
-                            alias($._var_comment, $.comment),
+                            $.comment,
                             field(
                                 "body",
                                 choice(
@@ -286,18 +295,12 @@ module.exports = grammar({
 
         raw_body: ($) =>
             seq(
-                choice(
-                    token(prec(1, seq(/.+/, NL))),
-                    seq(COMMENT_PREFIX, $._not_comment),
-                ),
+                token(prec(1, seq(/.+/, NL))),
                 optional($._raw_body),
             ),
         _raw_body: ($) =>
             seq(
-                choice(
-                    token(prec(PREC.RAW_BODY, LINE_TAIL)),
-                    seq(COMMENT_PREFIX, $._not_comment),
-                ),
+                token(prec(PREC.RAW_BODY, LINE_TAIL)),
                 optional($._raw_body),
             ),
         _not_comment: (_) => token(seq(/[^@]*/, NL)),
